### PR TITLE
Completed html-forms

### PR DESCRIPTION
**Pull request by MohamadAlsyouf**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Html Forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="get">
+      <h1>Coding Form!</h1>
+      <div>
+        <label>Email: <br /></label>
```
Comment (codingcodymiller) [inline html-forms/index.html:10]: This `label` needs to be configured with a `for` attribute that matches the `id` attribute on its form control element.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <input type="text" name="Email" />
+      </div>
+      <div>
+        <label>Why is coding cool? <br /></label>
```
Comment (codingcodymiller) [inline html-forms/index.html:14]: This `label` needs to be configured with a `for` attribute that matches the `id` attribute on its form control element.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          <option value="Because it's interesting">
+            Because it's interesting
+          </option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label>Favorite Language (choose one) <br /></label>
+        <input type="radio" name="language" value="HTML" />HTML
```
Comment (codingcodymiller) [inline html-forms/index.html:28]: This text labelling the `input` should be in a `label` element.

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        <input type="radio" name="language" value="CSS" />CSS
```
Comment (codingcodymiller) [inline html-forms/index.html:29]: This text labelling the `input` should be in a `label` element.

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        <input type="radio" name="language" value="JavaScript" />JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:30]: This text labelling the `input` should be in a `label` element.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+      <div>
+        <label>Would you like to sign up for more information? <br /></label>
+        <input type="checkbox" name="signup" value="Sign me up!" /> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:34]: This text labelling the `input` should be in a `label` element.

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+      <div>
+        <label>Comments <br /></label>
```
Comment (codingcodymiller) [inline html-forms/index.html:37]: This `label` needs to be configured with a `for` attribute that matches the `id` attribute on its form control element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by MohamadAlsyouf** [comment]
image

File: html-forms/index.html [hunk 8]
```diff
@@ -7,35 +7,50 @@
...
       <div>
-        <label>Favorite Language (choose one) <br /></label>
-        <input type="radio" name="language" value="HTML" />HTML
-        <input type="radio" name="language" value="CSS" />CSS
-        <input type="radio" name="language" value="JavaScript" />JavaScript
+        <label
+          >Favorite Language (choose one) <br />
+          <input type="radio" name="language" value="HTML" />HTML
+          <input type="radio" name="language" value="CSS" />CSS
+          <input type="radio" name="language" value="JavaScript" />JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: Each of these pieces of text needs to be in its own individual label so that it can pair with its form control element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by MohamadAlsyouf** [comment]
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**